config/rejection_reason_codes.yml

```diff
@@ -0,0 +1,55 @@
+---
+R01:
+  :id: qualifications
```

Comment:
is `:id` a necessary thing?

Reply:
I'm not convinced we do. 
We inflate these into `RejectionReasons::Reason` instances which all have an `id` but I can't find a dependency on that other than rendering the R4R wizard / form and we can never (re) edit existing rejection reasons in the wizard/form.